core/arch/riscv/kernel/abort.c

```diff
@@ -0,0 +1,394 @@
+// SPDX-License-Identifier: BSD-2-Clause
+/*
+ * Copyright 2022-2023 NXP
```

Comment:
When you copy as much as you do from arch/arm, then I think you should keep the original copyright too.